### PR TITLE
V1.8

### PR DESCRIPTION
**Pull request by kistitan**
Huge pull request (sorry)
- Restructure
- Refactor
- Code cleanup
- Comments
- Feed (model + service)

### CODE REVIEW

File: FetLife/fetlife/src/main/java/com/bitlove/fetlife/view/adapter/SwipeableRecyclerAdapter.java [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+                if (actionState == ItemTouchHelper.ACTION_STATE_SWIPE && isCurrentlyActive) {
+                    if (dX > 0) {
+                        setSwipeRightBackgroundVisibility(swipeableViewHolder,View.VISIBLE);
+                        setSwipeLeftBackgroundVisibility(swipeableViewHolder,View.GONE);
+                    } else if (dX < 0) {
+                        setSwipeRightBackgroundVisibility(swipeableViewHolder,View.GONE);
+                        setSwipeLeftBackgroundVisibility(swipeableViewHolder,View.VISIBLE);
+                    } else {
+                        setSwipeRightBackgroundVisibility(swipeableViewHolder,View.GONE);
```
Comment (reneklacan) [inline FetLife/fetlife/src/main/java/com/bitlove/fetlife/view/adapter/SwipeableRecyclerAdapter.java]: seems like you are missing space after comma :D

File: FetLife/fetlife/src/main/java/com/bitlove/fetlife/view/widget/WorkaroundTextView.java [hunk 2]
```diff
@@ -8,6 +8,9 @@
 import android.view.MotionEvent;
 import android.widget.TextView;
 
+/**
+ * Custom TextView to workaround issues of some Android custom OS bugs occurs when a text is selected within a scrollabe view
```
Comment (philippt) [inline FetLife/fetlife/src/main/java/com/bitlove/fetlife/view/widget/WorkaroundTextView.java]: "bugs that occur when a text is selected within a scrollable view"?

File: FetLife/fetlife/src/main/java/com/bitlove/fetlife/model/inmemory/InMemoryStorage.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+import android.util.SparseArray;
+
+import com.bitlove.fetlife.model.pojos.Story;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Store class for resource objects that should nto into the Database
```
Comment (BarryFruitman) [inline FetLife/fetlife/src/main/java/com/bitlove/fetlife/model/inmemory/InMemoryStorage.java]: "nto" -> "go"

**Comment by kistitan** [comment]
Pull request is pending as more changes coming up soon, that will make it obsoloate.
It is kept open for a while as some review comments still need to be followed.

**Comment by kistitan** [comment]
fixes done...
New MR is coming

**Pull request closed without merging**